readme.md

```diff
@@ -201,6 +201,10 @@ Explicit 256/Truecolor mode can be enabled using the `--color=256` and `--color=
 
 `chalkStderr` contains a separate instance configured with color support detected for `stderr` stream instead of `stdout`. Override rules from `supportsColor` apply to this too. `supportsColorStderr` is exposed for convenience.
 
+### modifiers, foregroundColors, backgroundColors, and colors
+
+All supported style strings are exposed for convenience. `colors` is the combination of `foregroundColors` and `backgroundColors`.
+
```

Comment:
I think an example could be useful, as it's not clear what these properties contain or what to use them for.